### ISSUE TITLE
Background and foreground colours do not have sufficent contrast ratio

### ISSUE THREAD

**Issue by a20behta** [HGustavs/LenaSYS#11784]
related to issue #9800 
the front page does not have enough contrast between each individual course. 
![image](https://user-images.githubusercontent.com/102468604/165053935-6e1e5f59-f2a4-4d78-a6f3-c02a84aceff1.png)

increase contrast on courses to make it easier to read
